src/metabase/cache_database.clj

```diff
@@ -0,0 +1,76 @@
+(ns metabase.cache-database
```

Comment:
yeah. it shouldn't be a top-level namespace either IMO. It should be `metabase.sync.something`. That way it's clear it's part of the sync part of the app.

@thearthur think about that chart you drew up showing how the different parts of the app interconnect, and how much more confusing that would have been to make if everything was just a top-level namespace